go/exorelay/exorelay_test.go

```diff
@@ -19,13 +20,12 @@ func FeatureContext(s *godog.Suite) {
 	exocom := exocomMock.New()
 	go exocom.Listen(4100)
 
-	s.Step(`^an ExoRelay with the configuration:$`, func(configStr *gherkin.DocString) error {
-		var config map[string]interface{}
-		err := json.Unmarshal([]byte(configStr.Content), &config)
-		exoInstance = New("ws://localhost:4100", config)
-		if err != nil {
-			return err
-		}
+	s.Step(`^an ExoRelay with the role "([^"]*)"$`, func(role string) error {
+		exoInstance = New(ExoRelayConfig{
```

Comment:
Sorry I meant `exorelay_test`. I have created https://github.com/Originate/exocom/pull/103 to address this comment, so please go ahead and ship this as is. Thanks!